data/json/items/comestibles/vegetable.json

```diff
@@ -0,0 +1,312 @@
+    "weight": 80,
+    "volume": 1,
+    "price": 200,
+    "charges": 6,
+    "nutrition": 5,
+    "healthy": 1,
+    "fun": -1,
+    "proportional": { "vitamins": [ "VitA", 3.0 ] },
+    "relative": { "spoils_in": 0.5 }
+  },
+  {
+    "id": "cattail_rhizome",
+    "copy-from": "vegetable",
+    "type": "COMESTIBLE",
+    "name": "cattail rhizome",
+    "name_plural": "cattail rhizomes",
+    "description": "A stout branching rhizome from a cattail plant.  Its crisp white flesh is very starchy and fibrous, but you really ought to cook it before you attempt to eat it.",
+    "weight": 418,
+    "volume": 2,
+    "price": 0,
+    "spoils_in": 80,
+    "nutrition": 22,
+    "fun": -12
+  },
+  {
+    "id": "cattail_stalk",
+    "copy-from": "vegetable",
+    "type": "COMESTIBLE",
+    "name": "cattail stalk",
+    "name_plural": "cattail stalks",
+    "description": "A stiff green stalk from a cattail plant.  It is starchy and fibrous, but it would be much better if you cooked it.",
+    "weight": 146,
+    "volume": 1,
+    "price": 0,
+    "spoils_in": 80,
+    "nutrition": 5,
+    "fun": -8
+  },
+  {
+    "id": "celery",
+    "copy-from": "vegetable",
+    "type": "COMESTIBLE",
+    "name": "celery",
+    "description": "Neither tasty nor very nutritious, but it goes well with salad.",
+    "weight": 80,
+    "volume": 1,
+    "price": 200,
+    "spoils_in": 150,
+    "nutrition": 5,
+    "quench": 3
+  },
+  {
+    "id": "cucumber",
+    "copy-from": "vegetable",
+    "type": "COMESTIBLE",
+    "name": "cucumber",
+    "description": "Come from the gourd family, not tasty but very juicy.",
+    "weight": 150,
+    "volume": 1,
+    "price": 300,
+    "spoils_in": 150,
+    "nutrition": 11,
+    "quench": 7,
+    "healthy": 1
+  },
+  {
+    "id": "garlic",
+    "copy-from": "vegetable_root",
+    "type": "COMESTIBLE",
+    "name": "garlic bulb",
+    "description": "A pungent garlic bulb.  Popular as a seasoning for its strong flavor.",
+    "weight": 120,
+    "volume": 1,
+    "price": 600,
+    "nutrition": 18,
+    "quench": -18,
+    "healthy": 1,
+    "fun": -18
+  },
+  {
+    "id": "lettuce",
+    "copy-from": "vegetable_leafy",
+    "type": "COMESTIBLE",
+    "name": "lettuce",
+    "description": "A crisp head of iceberg lettuce.",
+    "weight": 224,
+    "volume": 3,
+    "price": 100,
+    "charges": 4,
+    "spoils_in": 240,
+    "nutrition": 13,
+    "quench": 2
+  },
+  {
+    "id": "mushroom",
+    "copy-from": "vegetable",
+    "type": "COMESTIBLE",
+    "name": "mushroom",
+    "description": "Mushrooms are tasty, but be careful.  Some can poison you, while others are hallucinogenic.",
+    "weight": 159,
+    "volume": 1,
+    "price": 140,
+    "spoils_in": 48,
+    "nutrition": 14,
+    "healthy": 1,
+    "fun": -3
+  },
+  {
+    "id": "mushroom_magic",
+    "copy-from": "mushroom",
+    "type": "COMESTIBLE",
+    "extend": { "flags" : "HIDDEN_HALLU" }
```

Comment:
Note: HIDDEN_HALLU/POISON themselves do NOT cause poison or hallucinations. iuse functions do that.